tests/purs/failing/MultipleErrorsInOneRow.purs

```diff
@@ -0,0 +1,8 @@
+-- @shouldFailWith TypesDoNotUnify
+-- @shouldFailWith TypesDoNotUnify
+module A where
+
+-- Find both these errors
+f :: { a :: String, b :: String } -> Int
+f { a: 0, b: 0 } = 0
```

Comment:
Could you make the expected and actual types between `a` and `b` different, so that the two error messages are distinct?

Reply:
Done.